torch/cuda/__init__.py

```diff
@@ -23,6 +23,8 @@
 _original_pid = False
 _cudart = None
 
+CUDA_WINDOWS_LIB = 'cudart64_80'
```

Comment:
nit: is there a more generic way to do this, instead of relying on specific versions?